aiopylgtv/endpoints.py

```diff
@@ -1,42 +1,42 @@
-EP_GET_SERVICES = "api/getServiceList"
-EP_SET_MUTE = "audio/setMute"
-EP_GET_AUDIO_STATUS = "audio/getStatus"
-EP_GET_VOLUME = "audio/getVolume"
-EP_SET_VOLUME = "audio/setVolume"
-EP_VOLUME_UP = "audio/volumeUp"
-EP_VOLUME_DOWN= "audio/volumeDown"
-EP_GET_CURRENT_APP_INFO = "com.webos.applicationManager/getForegroundAppInfo"
-EP_LAUNCH_APP = "com.webos.applicationManager/launch"
-EP_GET_APPS = "com.webos.applicationManager/listLaunchPoints"
-EP_GET_APP_STATUS = "com.webos.service.appstatus/getAppStatus"
-EP_SEND_ENTER = "com.webos.service.ime/sendEnterKey"
-EP_SEND_DELETE = "com.webos.service.ime/deleteCharacters"
-EP_3D_ON = "com.webos.service.tv.display/set3DOn"
-EP_3D_OFF = "com.webos.service.tv.display/set3DOff"
-EP_GET_SOFTWARE_INFO = "com.webos.service.update/getCurrentSWInformation"
-EP_MEDIA_PLAY = "media.controls/play"
-EP_MEDIA_STOP = "media.controls/stop"
-EP_MEDIA_PAUSE = "media.controls/pause"
-EP_MEDIA_REWIND = "media.controls/rewind"
-EP_MEDIA_FAST_FORWARD = "media.controls/fastForward"
-EP_MEDIA_CLOSE = "media.viewer/close"
-EP_POWER_OFF = "system/turnOff"
-EP_POWER_ON = "system/turnOn"
-EP_SHOW_MESSAGE = "system.notifications/createToast"
-EP_LAUNCHER_CLOSE = "system.launcher/close"
-EP_GET_APP_STATE = "system.launcher/getAppState"
-EP_GET_SYSTEM_INFO = "system/getSystemInfo"
-EP_LAUNCH = "system.launcher/launch"
-EP_OPEN = "system.launcher/open"
-EP_GET_SYSTEM_SETTINGS = "settings/getSystemSettings"
-EP_TV_CHANNEL_DOWN = "tv/channelDown"
-EP_TV_CHANNEL_UP = "tv/channelUp"
-EP_GET_TV_CHANNELS = "tv/getChannelList"
-EP_GET_CHANNEL_INFO = "tv/getChannelProgramInfo"
-EP_GET_CURRENT_CHANNEL = "tv/getCurrentChannel"
-EP_GET_INPUTS = "tv/getExternalInputList"
-EP_SET_CHANNEL = "tv/openChannel"
-EP_SET_INPUT = "tv/switchInput"
-EP_CLOSE_WEB_APP = "webapp/closeWebApp"
-EP_INPUT_SOCKET = "com.webos.service.networkinput/getPointerInputSocket"
-EP_CALIBRATION = "externalpq/setExternalPqData"
+GET_SERVICES = "api/getServiceList"
```

Comment:
I would prefer to keep "EP_" in the constant names for the moment, since I'm planning to add also a few other things to the list of constants and want to be able to distinguish them.

Reply:
I restored those changes!

I thought since I refer to them as `ep.ENDPOINT_HERE` now, this was clear enough ;)